src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/main.js

```diff
@@ -110,6 +110,9 @@ define([
             if (code == 1101) {
                 errorMessage = 'security.roles.error.non-unique';
             }
+            if (code == 409) {
```

Comment:
The error code should probably stay `1101`,  and when we do what I said in https://github.com/sulu/sulu/pull/2783/files#r74434350 it probably will 😃